core/util/src/main/scala/net/liftweb/util/Props.scala

```diff
@@ -328,5 +378,18 @@ object Props extends Logger {
         Map()
     }
   }
+
+  private var prepended:List[java.util.Map[Object, Object]] = Nil
+  private var appended:List[java.util.Map[Object, Object]] = Nil
+  private var interpolators:List[java.util.Map[Object, Object]] = Nil
```

Comment:
It seems strange to me as a Scala framework to use an instance of `java.util.Map` for being our Record of Truth™. What's the reasoning there?